poethepoet/config.py

```diff
@@ -112,6 +119,13 @@ def validate(self):
             )
         # Validate env value
         env = self.global_env
+        if env:
+            for key, value in env.items():
+                if not isinstance(value, str):
+                    raise PoeException(
+                        f"Value of {key!r} in option `env` should be a string, but found {type(value)!r}"
+                    )
+        env = self.envfile
```

Comment:
I'm not sure this is useful as it is here. Parsed values will always be of type string. Also the error message would be wrong (refers to the env option), and if the file doesn't exist we'll get an unhandled exception.

Reply:
Oh right. I'll get that fixed

Reply:
> if the file doesn't exist we'll get an unhandled exception.

Fixed that in the latest commit.

Not sure if we should raise and error if it doesn't exist.

Reply:
Ideally the validate method should return error messages rather than raising errors.